src/chrome/content/rules/dlvr.it.xml

```diff
@@ -17,7 +12,6 @@
 		<test url="http://api.dlvr.it/1/help" />
```

Comment:
Move tests after the rule they apply to.

Reply:
I don't know what you're referencing as there are only four tests and they are all directly following their target.